swtbot_test/xml/testcase.xml

```diff
@@ -104,19 +104,39 @@
 	<TC id="TC7">
 		<project id="pg">
 			<group id="toolchain">
-				<toolchain>GCC</toolchain>
+				<!--toolchain>GCC</toolchain-->
```

Comment:
Q: Why not create new test case instead of modify existing one?

Reply:
After discussion with Hung Le, After this commit, in the future we don't have to commit this file, since this file is for testing data, and it change frequenly based on the needs at the time, therefore we can ignore it